### PR TITLE
Run post_remediation - quote group_names

### PR DESCRIPTION
**Pull request by diepes**
**Overall Review of Changes:**
Where run_audit.sh is called from ansible quote the group_name, as it can be a list containing a space.

To resolve a error

**Issue Fixes:**
Please list (using linking) any open issues this PR addresses
N/A
Resolves issue where there is more than one group, and the -g options fails due to space in command.
`/opt/UBUNTU20-CIS-Audit/run_audit.sh  ...  -g ['all_dev', 'webservers_dev']`

**Enhancements:**
Please list any enhancements/features that are not open issue tickets

**How has this been tested?:**
- fixed issue in local env that happens when a host belongs to more than one group.

### CODE REVIEW

**Review by uk-bolly — DISMISSED** [review]
great catch

**Comment by uk-bolly** [comment]
hi @diepes 

This is a great PR and good catch, unfortunatley the PR is not GPG signed so i am unable to accept it. GPG signature and sign off are both required.
Happy to merge if you can resolve this issue.

Many thanks

uk-bolly

**Comment by diepes** [comment]
hi @uk-bolly i added the signature.

Cheers,
diepes

**Comment by uk-bolly** [comment]
> hi @uk-bolly I added the signature.
> 
> Cheers, diepes

Hi @diepes 

Thanks for following up unfortunetly as you can see from the actions it is still failing with unverified.
The error i am seeing with the signature is
```The email in this signature doesn’t match the committer email.```

many thanks
uk-bolly

**Comment by diepes** [comment]
hi @uk-bolly , 
I eventually got the gpg and signoff aligned and the Verified icon.

image

**Review by uk-bolly — APPROVED** [review]

**Pull request merged**
